covsirphy/cleaning/jhu_data.py

```diff
@@ -119,7 +119,7 @@ def _cleaning(self):
         df.loc[df[self.COUNTRY] == "Diamond Princess", [
             self.COUNTRY, self.PROVINCE]] = ["Others", "Diamond Princess"]
         # Values
-        df = df.fillna(method="ffill").fillna(0)
+        df = df.fillna(0)
```

Comment:
I think `.fillna(method="ffill")` is required because `x` values in the following data should not be zeros.

0, 0, 1, 1, 2, 3,...10, 12, x, 16,...

Reply:
For the raw dataset this caused problem (not zero values there, but missing were as NaN) as it kept the previous country's values to the first missing of the next one's. I will revert this change too, since I will keep the raw=False dataset.